### DISCUSSION TITLE
日志文件删除周期失效

### DISCUSSION THREAD

**Discussion by zwac00001** [zuihou/zuihou-admin-cloud#267] [02.提问]
日志文件删除周期失效

**Comment by zuihou** [comment]
最新版本应该修复了这个问题

**Reply by zwac00001** [reply]
最新版

**Comment by zwac00001** [comment]
好的，日志每天归一档也没有生效

**Reply by zuihou** [reply]
归档规则是： 单个日志文件最多 100MB, 60天的日志周期，最大不能超过20GB

可以按自己的要求修改 appender 

<img width="1573" alt="image" src="https://user-images.githubusercontent.com/8500410/230540860-4bfa1e8e-bf9a-43eb-83bd-3a255bea5603.png">
